src/components/PhotoPage/PhotoPage.js

```diff
@@ -108,21 +100,44 @@ const officers = [
 	},
 	{
 		role: 'Officer',
-		name: 'Furn Techalertumpai',
-		id: 'furn',
-		description: `Hi! I'm a fourth year CS major from Bangkok, Thailand 
-		(you can tell from my very long last name). I am a BIG fan of food and 
-		I like to listen to new jazz/R&B music. I will be co-directing Hack 
-		Kitchen with Raji this quarter so come say hi!!`
-	},
-	{
-		role: 'Officer',
-		name: 'Galen Wong',
+		name: 'Galen-senpai',
 		id: 'galen',
 		description: `I teach Web Dev a lot, feel free to ask me questions on 
 		JavaScript and React. I randomly organized a hot pot party at Kevin's 
 		place and they seem pretty satisfied.`
-	}
+	},
+	{
+		role: 'Officer',
+		name: 'Miles Wu',
+		id: 'miles',
+		description: `I’m a CS major passionate about hackathons and Powell library.
+		 Most of my experience deals with web/mobile dev. Catch me out here eating way too much food, drinking way too 
+		 much boba (@Jamie for enabling me), and vibing to chill music.`
+	},
+	{
+		role: 'Officer',
+		name: 'Alex Xia',
+		id: 'alex',
+		description: `Hi, I'm a first year CS major from Hong Kong. Aside from coding, 
+		I like to watch anime and play games - occasionally, Jody baits me into watching kdrama. 
+		When I am not doing these things, I am at the UCLA badminton club.`
+	},
+	{
+		role: 'Advisor',
+		name: 'Lea Blum',
+		id: 'lea',
+		description: `
+		Hi there, I’m Lea. Come hang out with me and the rest of hack at our events (on zoom) to hear my 
+		(sometimes) funny jokes and (always) hilarious puns. Currently living off of Disney movies in quarantine.
+		`
+	},
+	{
+		role: 'Advisor',
+		name: 'Connie Chen',
+		id: 'connie',
+		description: `Hello! My name is Connie, and I'm a third year Computer Science major and Cognitive Science minor. 
+		I like the California weather, messenger stickers, and music. Currently living off of Disney movies in quarantine.`
```

Comment:
```suggestion
		I like the California weather, Messenger stickers, and music. Currently living off of Disney movies in quarantine.`
```

Also the "Currently living off of Disney movies in quarantine." seems to be copy-pasted from Lea? It doesn't like Connie lol.

Reply:
This is what they both actually submitted for their bios :)

Reply:
Heh ok…